velox/exec/tests/utils/AssertQueryBuilder.h

```diff
@@ -114,6 +114,12 @@ class AssertQueryBuilder {
   std::shared_ptr<Task> assertResults(
       const std::vector<RowVectorPtr>& expected);
 
+  /// Run the query and ensure it returns batches of `expectedType`, and exactly
+  /// `numRows`.
+  std::shared_ptr<Task> assertNumRows(
```

Comment:
nit: Since this function checks type in addition to the number of rows, maybe we can rename it as `assertTypeAndNumRows`?